packages/flutter/test/material/paginated_data_table_test.dart

```diff
@@ -1022,6 +1022,39 @@ void main() {
     await binding.setSurfaceSize(originalSize);
   });
 
+  testWidgets('dataRowMinHeight & dataRowMaxHeight if not set will use DataTableTheme', (WidgetTester tester) async {
+    final Size originalSize = binding.renderView.size;
+    await binding.setSurfaceSize(const Size(800, 800));
+
+    const double minMaxDataRowHeight = 30.0;
+
+    Widget buildTable() {
+      return MaterialApp(
+        theme: ThemeData.light().copyWith(
+          dataTableTheme: const DataTableThemeData(
+            dataRowMinHeight: minMaxDataRowHeight,
+            dataRowMaxHeight: minMaxDataRowHeight,
+          ),
+        ),
+        home: PaginatedDataTable(
+          header: const Text('Test table'),
+          source: TestDataSource(allowSelection: true),
+          columns: const <DataColumn>[
+            DataColumn(label: Text('Name')),
+            DataColumn(label: Text('Calories'), numeric: true),
+            DataColumn(label: Text('Generation')),
+          ],
+        ),
+      );
+    }
+
+    await tester.pumpWidget(buildTable());
+    expect(tester.getSize(find.widgetWithText(Container, 'Frozen yogurt (0)').first).height, minMaxDataRowHeight);
```

Comment:
```suggestion
    await tester.pumpWidget(MaterialApp(
      theme: ThemeData(
        dataTableTheme: const DataTableThemeData(
          dataRowMinHeight: minMaxDataRowHeight,
          dataRowMaxHeight: minMaxDataRowHeight,
        ),
      ),
      home: PaginatedDataTable(
        header: const Text('Test table'),
        source: TestDataSource(allowSelection: true),
        columns: const <DataColumn>[
          DataColumn(label: Text('Name')),
          DataColumn(label: Text('Calories'), numeric: true),
          DataColumn(label: Text('Generation')),
        ],
      ),
    ));

    final Container rowContainer = tester.widget<Container>(find.descendant(
      of: find.byType(Table),
      matching: find.byType(Container),
    ).last);
    expect(rowContainer.constraints?.minHeight, minMaxDataRowHeight);
    expect(rowContainer.constraints?.maxHeight, minMaxDataRowHeight);
```

Reply:
I agree this is better, thank you for the recommendation